packages/pillow_heif/meta.yaml

```diff
@@ -1,9 +1,9 @@
 package:
-  name: pillow_heif
-  version: 0.8.0
+  name: pillow-heif
```

Comment:
Thanks for the catch. Could you also update the directory name? (pillow_heif -> pillow_heif)

Reply:
>  Could you also update the directory name? (pillow_heif -> pillow_heif)

🤔 I think we had to rename the package and directory names to `pillow-heif` (maybe there was a typo here)?

Reply:
Yes, that's what I meant, but I made a typo 😅